.github/workflows/dev.yml

```diff
@@ -274,6 +274,31 @@ jobs:
       - name: Test
         run: cd build && ctest -j3 --output-on-failure
 
+  bazel:
+    # Tests with: Bazel build system
+    name: Bazel
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup
+        run: |
+          sudo apt-get -qq install apt-transport-https curl gnupg -y
+          curl -fsSL https://bazel.build/bazel-release.pub.gpg | gpg --dearmor >bazel-archive-keyring.gpg
+          sudo mv bazel-archive-keyring.gpg /usr/share/keyrings
+          echo "deb [arch=amd64 signed-by=/usr/share/keyrings/bazel-archive-keyring.gpg] https://storage.googleapis.com/bazel-apt stable jdk1.8" | sudo tee /etc/apt/sources.list.d/bazel.list
+
+          sudo apt-get -qq update && sudo apt-get -qq install bazel
```

Comment:
[`bazelisk`](https://github.com/bazelbuild/bazelisk) should be available on all github runners, no need to install! Usually I add a `.bazelversion` to the root of the repository and bazelisk will use that, but the [`USE_BAZEL_VERSION`](https://github.com/bazelbuild/bazelisk?tab=readme-ov-file#how-does-bazelisk-know-which-bazel-version-to-run) env var also works.

Reply:
OK, that's good to know. I've switched to using `baselisk`, which seems to work fine. It loads v8. I haven't pinned that version, we can just let it auto-update.

Alternatively, if you want to pin a specific version, it will be your job (or someone else) to keep it updated, rather than me.

Reply:
Using latest should be fine!